walletlibrarydemo/src/main/java/com/microsoft/walletlibrarydemo/feature/issuance/presentationlogic/ViewModelFactory.kt

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.walletlibrarydemo.feature.issuance.presentationlogic
+
+import android.content.Context
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+
+class ViewModelFactory(
+    private val context: Context
+): ViewModelProvider.NewInstanceFactory() {
+    override fun <T : ViewModel> create(modelClass: Class<T>): T = SampleViewModel(context) as T
```

Comment:
do we need a ViewModelFactory? Isnt it just one view model?

Reply:
If a ViewModel requires a parameter and we don't have dependency injection, it has to be done this way.